dcos/marathon.py

```diff
@@ -152,8 +203,7 @@ def get_about(self):
         :rtype: dict
         """
 
-        url = self._create_url('v2/info')
-        response = _http_req(http.get, url, timeout=self._timeout)
```

Comment:
All of the changes to existing methods in this class are the same: removal of the URL construction and timeout handling code, which is now taken care of by the `RpcClient`.